mm2src/coins/utxo/utxo_common.rs

```diff
@@ -2673,13 +2673,19 @@ where
         script_pubkey: op_return_script,
     };
 
+    let addr_format = match coin.as_ref().my_address.addr_format.clone() {
```

Comment:
Do I understand correctly that we can safely replace it with `UtxoAddressFormat::Standard`? Even if the cash format is activated we can still use `Standard` as they are just different representations of the same address.

Reply:
Yes, that should be no problem

Reply:
I've also noticed that it might cause problems in the future if we add a new address format: it will be automatically assigned to the payment address here causing the same bug. Could you change it, please? :slightly_smiling_face: